src/bindings/c/docs/How_to_Create_C_Sample_with_OpenVINO_C_API.md

```diff
@@ -0,0 +1,74 @@
+# How to Create C Sample with OpenVINO C API
```

Comment:
What is the purpose of this guide for the developer? From my point of view developer documentation should demonstrate real often cases how developer can add something to component.

Moreover sample is not a part of C API